src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_image.py

```diff
@@ -38,10 +38,10 @@ def test_read_images_from_alias_doc(self):
         # assert
         win_images = [i for i in images if i['publisher'] == 'MicrosoftWindowsServer']
         self.assertTrue(len(win_images) > 0)
-        ubuntu_image = next(i for i in images if i['publisher'] == 'Canonical')
+        ubuntu_image = next(i for i in images if i['publisher'] == 'Canonical' and i['sku'] == '22_04-lts-gen2')
```

Comment:
`unittest.TestCase` should only be used for small and light-weight tests. `list_vm_images` is a heavy operation and requires network access. Its result is also not stable. It should be put inside a `LiveScenarioTest` if no recording should be generated.

Reply:
@yanzhudd Please help refine this test when you have time, thanks~